web/src/portico/landing-page.ts

```diff
@@ -72,7 +72,7 @@ const apps_events = function (): void {
             description:
                 "The Zulip desktop app comes with native <a class='apps-page-link' href='/help/desktop-notifications'>desktop notifications</a>, support for multiple Zulip accounts, and a dedicated tray icon.",
             download_link: "/apps/download/mac",
```

Comment:
This should switch to `/apps/download/mac-arm64`

Reply:
Done.